Docs/Configuration.tex

```diff
@@ -3412,46 +3439,49 @@ \subsection{Debug Properties}\label{miscdebugprops}
     \item \texttt{0x40} (bit \texttt{6}) --- Enable logging to file.
```

Comment:
Should non-volatile be also be guarded with `\texttt{}` here? Given the text below.